src/eligibility-check-opkit.ts

```diff
@@ -0,0 +1,195 @@
+              benefit.type === 'deductible' &&
+              benefit.period === 'remaining'
+          );
+
+          const filteredDeductible = benefits.find(
+            (benefit: any) =>
+              benefit.network === network &&
+              benefit.service === category &&
+              benefit.coverage === unit &&
+              benefit.type === 'deductible' &&
+              benefit.period === 'service_year'
+          );
+
+          if (filteredDeductible && filteredRemainingDeductible) {
+            fhirBenefit.push({
+              type: {
+                coding: [
+                  {
+                    code: 'deductible',
+                  },
+                ],
+              },
+              allowedMoney: {
+                value: filteredDeductible.values[0].value || 0,
+                currency: 'USD',
+              },
+              usedMoney: {
+                value: filteredRemainingDeductible.values[0].value || 0,
+                currency: 'USD',
+              },
+            });
+          }
+
+          item.push({
+            category: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/ex-benefitcategory',
+                  code: serviceToCategoryCodeMap[category],
+                  display: serviceToCategoryDisplayMap[category],
+                },
+              ],
+            },
+            network: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/benefit-network',
+                  code: networkToCode[network],
+                },
+              ],
+            },
+            unit: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/benefit-unit',
+                  code: unit,
+                },
+              ],
+            },
+            benefit: fhirBenefit,
+          });
+        });
+      });
+    });
+
+    return item;
+  };
+
+  const coverageEligibilityRequest = await medplum.createResource({
+    resourceType: 'CoverageEligibilityRequest',
+    identifier: [{
+        system: 'www.opkit.co/id', //You should choose this
+        value: 'opkit-identifier1234' //Ideally something like opkitRequest.subscriber?
+    }],
+    insurer: createReference(organization),
+    patient: createReference(patient),
+    insurance: [
+      {
+        focal: true,
+        coverage: createReference(coverage),
+        businessArrangement: 'opkit-identifier1234' //If there is some kind of contract number?
+      }
+    ],
+    item: generateItem(result)
+  });
+
+  const coverageEligibilityResponse = await medplum.createResource({
+    resourceType: 'CoverageEligibilityResponse',
+    status: 'active',
+    outcome: 'complete',
+    purpose: ['validation', 'benefits'],
+    request: createReference(coverageEligibilityRequest),
+    disposition: 'Policy is currently in-force.', //Won't this come from opkit?
```

Comment:
Yes! I originally just hardcoded this to see if I could properly create the `coverageEligibilityResponse`.

Here is the additional code I would add to check for active/inforce insurance:

```
const isPlanActive = (opkitResult.plan.benefits) => {
  const activeStatusBenefit = benefitFeatures.find(
    (benefit) =>
      benefit.type === "active_coverage" &&
      benefit.service === "health_benefit_plan_coverage",
  );

  return !!activeStatusBenefit;
};
```

And then we could do something like
```
disposition: isPlanActive ? "Policy is currently in-force" : "Policy is currently not in-force";